src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -950,7 +950,9 @@ bool HMICapabilitiesImpl::LoadCapabilitiesFromFile() {
             ConvertStringToEnum<hmi_apis::Common_HmiZoneCapabilities::eType>(
                 ui_hmi_zone_capabilities_node.asString());
         if (hmi_apis::Common_HmiZoneCapabilities::INVALID_ENUM != enum_value) {
-          smart_objects::SmartObject hmi_zone_capabilities_so = enum_value;
+          smart_objects::SmartObject hmi_zone_capabilities_so(
+              smart_objects::SmartType_Array);
+          hmi_zone_capabilities_so = enum_value;
```

Comment:
this change does not make sense, it is not an array. what is the cppcheck issue associated?

Reply:
It was array before. Compile error was caused by wrong merge(previous commit), so so I just fixed it in last commit.
Before cppcheck issues fix it was 
smart_objects::SmartObject hmi_zone_capabilities_so =
            smart_objects::SmartObject(smart_objects::SmartType_Array);
what caused redundant Initialization error

Reply:
I see now this is not a change by you, in commit view it looked like it was. I apologize!

Reply:
No problem))